### PR TITLE
Schedule

### PR DESCRIPTION
**Pull request by lboynton**
Umm yeah.

### CODE REVIEW

File: _data/speakers.yml [hunk 1]
```diff
@@ -0,0 +1,215 @@
...
+davies:
+  name: Oliver Davies
+  avatar: davies.jpg
+  twitter: opdavies
+  bio: |
+    Oliver Davies has been building websites since 2007, and specialising in PHP and Drupal since 2008. He is currently a Lead Developer at [CTI Digital](https://www.ctidigital.com), is a Drupal 7 and 8 core contributor, and a provisional member of the Drupal Security team. In Oliver's spare time, he enjoys spending time with his family, co-organises the [Drupal Bristol](http://www.drupalbristol.org.uk) and [PHPSW](https://phpsw.uk) user groups, and is a [DrupalCamp Bristol](http://www.drupalcampbristol.co.uk) committee member. You can follow him on Twitter at [@opdavies](https://twitter.com/opdavies).
+  talks:
+    - title: Drupal 8 - Rejoining the Herd
+      abstract: |
+        [Drupal 8](https://www.drupal.org/8) was (finally) released on November 19th 2015, after almost 4 years of work and code commits by over 3,200 different contributors. Whilst it's pretty much the same as the Drupal that we know and, hopefully, love, a lot has changed behind the scenes and under the hood!
```
Comment (lboynton) [inline _data/speakers.yml:65]: So if anyone wants to go through these and make them HTML links that would be good :)

File: schedule2.html [hunk 2]
```diff
@@ -0,0 +1,266 @@
+---
```
Comment (lboynton) [inline schedule2.html:1]: This version is more responsive, but it doesn't handle the lightning talks/main talk crossover very well (need something akin to rowspan here)

File: _data/speakers.yml (hunk 1 above)
Comment (asgrim) [inline _data/speakers.yml:65]: Does the markdown links not work? o_O

Comment (lboynton) [inline _data/speakers.yml:65]: No, maybe it can be enabled somehow though.

Comment (lboynton) [inline _data/speakers.yml:65]: markdownify :)

File: schedule.html [hunk 3]
```diff
@@ -0,0 +1,575 @@
...
+                <th>Atrium</th>
+            </tr>
+            </thead>
+            <tbody>
+            <tr>
+                <td>08:00</td>
+                <td>
+                    <strong>Registration &amp; refreshments</strong>
+                    <p>Please arrive between 8am and 9am to register. A light breakfast including tea, coffee and
+                        pastries will be available.</p>
```
Comment (lboynton) [inline schedule.html]: Not sure if this is true

File: schedule.html [hunk 4]
```diff
@@ -0,0 +1,575 @@
...
+                <th class="col-md-1"></th>
+                <th>Atrium</th>
+            </tr>
+            </thead>
+            <tbody>
+            <tr>
+                <td>10:50</td>
+                <td>
+                    <strong>Break &amp; refreshments</strong>
+                    <p>Tea and biccies!</p>
```
Comment (lboynton) [inline schedule.html]: Something better here perhaps

File: schedule.html [hunk 5]
```diff
@@ -0,0 +1,575 @@
...
+                <th class="col-md-1"></th>
+                <th>Atrium</th>
+            </tr>
+            </thead>
+            <tbody>
+            <tr>
+                <td>12:05</td>
+                <td>
+                    <strong>Lunch service</strong>
+                    <p>Some hot, hopefully nice food will be served. Maybe with peas or something.</p>
```
Comment (lboynton) [inline schedule.html]: And here ;)

File: schedule.html [hunk 6]
```diff
@@ -0,0 +1,575 @@
...
+                <th class="col-md-1"></th>
+                <th>Atrium</th>
+            </tr>
+            </thead>
+            <tbody>
+            <tr>
+                <td>14:00</td>
+                <td>
+                    <strong>Break &amp; refreshments</strong>
+                    <p>More tea and biccies!</p>
```
Comment (lboynton) [inline schedule.html]: and here

**Comment by rossey** [comment]
:+1:

**Comment by rossey** [comment]
See https://phpsouthcoast.slack.com/files/rossey/F0L053L5U/screencapture-127-0-0-1-4000-schedule-1454347434293.png for the screenshot

**Pull request merged**